src/style/pinny.scss

```diff
@@ -102,3 +105,21 @@
 [hidden] {
     display: none !important;
 }
+
+
+// Shade
+// -----
+//
+// Let shadow fill 100% of viewport background after rotation
+// on Android 4.3 (HTC, Samsung Galazy S3).
+//
+// Use !important to overwrite inline position styles
+
+.shade {
```

Comment:
I totally agree @scalvert and wanted to make it in JS but couldn't =(
Can we pass it to an engineer?